src/UI/BodyPanel.java

```diff
@@ -0,0 +1,233 @@
+            }
+        });
+        tableLeft.addFocusListener(new java.awt.event.FocusAdapter() {
+            public void focusGained(java.awt.event.FocusEvent evt) {
+                tableLeftFocusGained(evt);
+            }
+        });
+        jScrollPaneLeft.setViewportView(tableLeft);
+
+        jSplitPane.setLeftComponent(jScrollPaneLeft);
+
+        jScrollPaneRigth.addMouseListener(new java.awt.event.MouseAdapter() {
+            public void mouseClicked(java.awt.event.MouseEvent evt) {
+                jScrollPaneRigthMouseClicked(evt);
+            }
+        });
+
+        tableRigth.setModel(new javax.swing.table.DefaultTableModel(
+            new Object [][] {
+
+            },
+            new String [] {
+                "Name", "Type", "Size", "Date"
+            }
+        ) {
+            Class[] types = new Class [] {
+                java.lang.String.class, java.lang.String.class, java.lang.String.class, java.lang.String.class
+            };
+            boolean[] canEdit = new boolean [] {
+                false, false, false, false
+            };
+
+            public Class getColumnClass(int columnIndex) {
+                return types [columnIndex];
+            }
+
+            public boolean isCellEditable(int rowIndex, int columnIndex) {
+                return canEdit [columnIndex];
+            }
+        });
+        tableRigth.addFocusListener(new java.awt.event.FocusAdapter() {
+            public void focusGained(java.awt.event.FocusEvent evt) {
+                tableRigthFocusGained(evt);
+            }
+        });
+        jScrollPaneRigth.setViewportView(tableRigth);
+
+        jSplitPane.setRightComponent(jScrollPaneRigth);
+
+        this.add(jSplitPane, java.awt.BorderLayout.CENTER);
+    }
+    
+    /**
+     * Method to add the roots to panel
+     * @param evt 
+     */
+    public void formWindowOpened(java.awt.event.WindowEvent evt) {                                  
+        File[] drives = File.listRoots();
+        for (int index = 0; index < drives.length; index++) {
+            checkBoxSelectDrive.addItem(drives[index].getPath());
+        }
+    }                                 
+
+    /**
+     * Method to set the file path 
+     * @param evt 
+     */
+    private void checkBoxSelectDriveActionPerformed(java.awt.event.ActionEvent evt) {                                                    
+        filePath.setText(checkBoxSelectDrive.getSelectedItem().toString());
+    }                                                   
+
+    /**
+     * Method to list the file when the user selects Go button
+     * @param evt 
+     */
+    private void jButtonGoActionPerformed(java.awt.event.ActionEvent evt) {                                          
+        listFiles(filePath.getText());
+    }    
+    
+    /**
+     * Method to list all files of a given path 
+     * @param pathFile where the items will be retrieved 
+     */
+    public void listFiles(String pathFile) {
```

Comment:
the name does not match with the action that is doing, is not listing the files, in fact is loading the files into the panel, and besides this functionality should be using the one created by Daniel